bin/send_email.py

```diff
@@ -73,8 +73,9 @@ def email_template(
         (
             "VALIDATED",
             "yes",
-        ):
-        f"Submission {submission_id} has been evaluated with the following scores:\n" + "\n".join(get_score_dict(score)) + f"\nView all your scores here: https://www.synapse.org/#!Synapse:{view_id}/tables/",
+        ): f"Submission {submission_id} has been evaluated with the following scores:\n"
+        + "\n".join(get_score_dict(score))
+        + f"\nView all your scores here: https://www.synapse.org/#!Synapse:{view_id}/tables/",
```

Comment:
Will the `score` here exclude the `score_status` property in the output result from the scoring step?

Reply:
Yes, it just includes the submission ID and a list of the scores.